03-transactions.md

```diff
@@ -698,73 +698,408 @@ HTLC-success) gives a weight of:
+    # local_signature = 304402203390ec7939c2000500c5c859912acaa37811a9182b71f733c11fb30e62eb983602207f7c477f12093e99ba16e2b41bbcfe518b1ab02410789b83ad66a241b606513e
+    output commit_tx: 0200000000010142a26bb3a430a536cf9e3a8ce2cbcb0427c29ec6c7d647175cfc78328b57fba7010000000038b02b8002c0c62d0000000000160014e2f14ead9ca9a2f4c8b8a3f9bd109762ed33a03654a56a0000000000220020b736bcf56020e73439245b2d13ef024ac9d1eaf802616d82cedeae33ecd7acba040047304402203390ec7939c2000500c5c859912acaa37811a9182b71f733c11fb30e62eb983602207f7c477f12093e99ba16e2b41bbcfe518b1ab02410789b83ad66a241b606513e01483045022100ce5dadef1f831e531b0c07384c777b9bee6f7d712838579f432feb796cdd1470022057907a69b6d134a394d1450c3e92f53d99730a4818a06359a040f5656a6fb30001475221023da092f6980e58d2c037173180e9a465476026ee50f96695963e8efe436f54eb21030e9f7b623d2ccc7c9bd44d66d5ce21ce504c0acf6385a132cec6d3c39fa711c152ae3e195220
+    num_htlcs: 0
+    
+    name: commitment tx with all 5 htlcs untrimmed (minimum feerate)
+    to_local_msat: 6988000000
     to_remote_msat: 3000000000
-    htlc 0 direction: local->remote
-    htlc 0 amount_msat: 10000000
-    htlc 0 expiry: 443210
-    htlc 0 payment_hash: 4231f5cbd75de98283f65f0eadf95ba05cd7331c94d88934a69031fda244d70e
-    htlc 1 direction: remote->local
-    htlc 1 amount_msat: 20000000
-    htlc 1 expiry: 453203
-    htlc 1 payment_hash: a44748f172677f67585ea34acf38fdc8f7c566c34e176842f2d87d2afa16f240
-    output commit_tx: 020000000142a26bb3a430a536cf9e3a8ce2cbcb0427c29ec6c7d647175cfc78328b57fba70100000000534738800410270000000000002200207d8a3d7311cf89ac5e3110bf97a68fa69c0079cd0329d56135d5246638828202204e000000000000220020fca23ba30fe0400a01af3c1ca47ceeb679bff7bad703f0e44c7d3266faea88ccc0c62d00000000001600141844e52616af46f531635b5b770737ec5695a08b741d6a00000000002200201c1a9b14ca64510fa53ec4a910dfbf023983489f82f3f51c5884941ef0d1944234ee4e00
-    output htlc_timeout_tx 0: 02000000015ffa1b4fea9232e5ca2b705474dc261c5e93617ed75d284061b27abaaacb2f1f0000000000ffffffff01c9020000000000002200201c1a9b14ca64510fa53ec4a910dfbf023983489f82f3f51c5884941ef0d194424ac30600
-    output htlc_success_tx 0: 02000000015ffa1b4fea9232e5ca2b705474dc261c5e93617ed75d284061b27abaaacb2f1f0100000000ffffffff01bb270000000000002200201c1a9b14ca64510fa53ec4a910dfbf023983489f82f3f51c5884941ef0d1944200000000
-
-    name: with htlcs, some below dust limit
-    to_local_msat: 6839700000
+    local_feerate_per_kw: 0
+    # base commitment transaction fee = 0
+    # HTLC offered amount 2000 wscript 21030d417a46946384f88d5f3337267c5e579765875dc4daca813e21734b140639e77c820120876475527c21039390232673a9de88820d44ea910f364a332dc815cb0122bf5088d581dcbac87852ae67a914b43e1b38138a41b37f7cd9a1d274bc63e3a9b5d188ac68
+    # HTLC offered amount 3000 wscript 21030d417a46946384f88d5f3337267c5e579765875dc4daca813e21734b140639e77c820120876475527c21039390232673a9de88820d44ea910f364a332dc815cb0122bf5088d581dcbac87852ae67a9148a486ff2e31d6158bf39e2608864d63fefd09d5b88ac68
```

Comment:
Ah, thanks!  I decided to canonicalize arg order as "localkey, remotekey" and missed one :(